step.yml

```diff
@@ -42,3 +40,19 @@ inputs:
 
         - 2.8.6
         - latest
+  - allow_caching: "local"
+    opts:
+      title: Allow caching
+      description: |-
+        Allows caching for the step.
+
+        'local' enables the caching the packages in the build directory.
+        'global' enables the caching of the global-packages folder, this is where NuGet installs any downloaded package.
+        'none' disables the caching for the step.
+
+        Please find more information about caching here:
+        https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders
+      value_options:
+      - "local"
```

Comment:
have option to cache both local and global packages

Reply:
Added case. Note: updated Jira ticket, as this wasn't a requirement.

Reply:
pls add to the step.yml too.

Reply:
Added.